recipes/crispector/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: crispector
+  version: "1.0.0"
+
+build:
+  number: 0
+  entry_points:
+    - crispector = crispector.arg_parser:cli
+  noarch: generic
+
+source:
+  url: https://github.com/YakhiniGroup/crispector/archive/v1.0.0b.tar.gz
```

Comment:
Is it correct that the version for the archive (1.0.0b) is different from the package version (without the "b")?

Reply:
No. corrected.